front/src/service.ts

```diff
@@ -17,6 +17,8 @@
 // too often.
 
 import axios from 'axios';
+import { io, Socket } from "socket.io-client";
+const socket = io("http://localhost:5000/wsk");
```

Comment:
Avoid hardcoding the server url, there should be a url prop on ApiService.

Reply:
idk where i can put in service.ts
 ```js
const server = io ();
```

Reply:
Fixes here [b839b69](https://github.com/Draeggon/scratchy/pull/2/commits/b839b690cda474b9c595b3b63a8210257a37e800)